### PR TITLE
MINOR: Reorder the modifiers and Replace Map.get with Map.computeIfAbsent

### PR DESCRIPTION
**Pull request by tang7526**
- Reorder the modifiers to comply with the Java Language Specification.
- Replace Map.get() and condition with a call to Map.computeIfAbsent().

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java [hunk 1]
```diff
@@ -644,11 +644,7 @@ private UnsentRequests() {
         public void put(Node node, ClientRequest request) {
             // the lock protects the put from a concurrent removal of the queue for the node
             synchronized (unsent) {
-                ConcurrentLinkedQueue<ClientRequest> requests = unsent.get(node);
-                if (requests == null) {
-                    requests = new ConcurrentLinkedQueue<>();
-                    unsent.put(node, requests);
```
Comment (chia7712) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java:650]: You have to call ```put(node, requests)```

**Review by chia7712 — COMMENTED** [review]
@tang7526 Thanks for your patch. one major comment is left.

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java (hunk 1 above)
Comment (chia7712) [reply clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java:650]: my bad. please ignore above comment :(

**Review by chia7712 — APPROVED** [review]
LGTM

**Comment by chia7712** [comment]
@tang7526 Could you merge trunk to trigger QA again?

**Comment by tang7526** [comment]
> @tang7526 Could you merge trunk to trigger QA again?

Done

**Pull request merged**